datascience-notebook/Dockerfile

```diff
@@ -12,3 +12,6 @@ COPY ./files/apt_packages /tmp/
 
 RUN conda install boto3 \
     && apt-get update && apt-get install -y $(cat /tmp/apt_packages)
+
+# Fix conda install for NB_UID
+RUN chown -R $NB_UID /opt/conda
```

Comment:
Looks good to me 👍 

Do you know if we need to make the same change to [the other `Dockerfile` (all-spark-notebook)](https://github.com/ministryofjustice/analytics-platform-jupyter-notebook/blob/master/all-spark-notebook/Dockerfile)?

Reply:
good point, I guess both of them need it